src/components/LandingPage.tsx

```diff
@@ -1,40 +1,93 @@
-import { Box, Button, HStack, Link, VStack } from '@navikt/ds-react'
+'use client'
+
+import {
+  Box,
+  Button,
+  Heading,
+  HStack,
+  Link,
+  VStack,
+  BodyLong,
+  List,
+} from '@navikt/ds-react'
+import stepStyles from './styles/stepStyles.module.css'
 import React from 'react'
+import Substep from './Substep'
 
-export default function QuestionBox() {
-  return (
-    <Box width={'full'} background="surface-subtle">
-      <div className="flex flex-col items-center w-full">
-        <h2 className=" mb-3">Pensjonskalkulator</h2>
+export default function LandingPage() {
+  const Icon = ({ color }: { color: string }) => (
+    <Box
+      width="1rem"
+      height="1rem"
+      style={{ background: color }}
+      borderRadius="full"
+      aria-hidden
+    ></Box>
+  )
 
-        <Box
-          marginBlock={'auto'}
-          width={'100%'}
-          maxWidth={'40rem'}
-          padding={'4'}
-          background="bg-default"
-        >
-          <VStack padding={'6'} gap={'5'}>
-            <h3>Hei Du</h3>
-            <p>Velkommen til pensjonskalkulatoren som kan vise deg:</p>
-            <div>
-              <p>Alderspensjon (NAV)</p>
-              <p>AFP (avtalefestet pensjon)</p>
-              <p>Pensjonsavtaler</p>
-            </div>
-            <p>
-              For å beregne pensjonen din, må du svare på alle spørsmålene som
-              kommer.
-            </p>
-            <HStack gap={'2'}>
-              <Link href="./kalkulator-uinnlogget/form" className="text-white">
-                <Button variant="primary">Kom i gang</Button>
-              </Link>
+  return (
```

Comment:
Testen ser ikke ut til å være utvidet til å ta høyde for disse endringene.
Sjekk noe av innholdet og sjekk klikk på buttons/links